DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -241,6 +265,29 @@ OUTPUT Inserted.AdminID
                 values
             );
 
+            var userCentreDetailsValues = new
+            {
+                adminUserId,
+                CentreId = registrationModel.Centre,
+                Email = registrationModel.SecondaryEmail,
+            };
+
+            connection.Execute(
+                @"INSERT INTO UserCentreDetails
+                    (
+                        UserId,
+                        CentreId,
+                        Email
+                    )
+                    VALUES
+                    (
+                        @userId,
+                        @centreId,
+                        @email
+                    )",
+                userCentreDetailsValues
+            );
+

```

Comment:
This needs a few changes, but it requires the UserId to be passed into the RegisterAdmin method. Jon is doing this in https://github.com/TechnologyEnhancedLearning/DLSV2/pull/1110/files, so I think it's best if you remove the changes to this method and leave it all for him to sort out after you merge.